nephio/optional/nephio-stock-repos/repo-oai-core-packages.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: config.porch.kpt.dev/v1alpha1
+kind: Repository
+metadata:
+  name: catalog-workloads-oai-core
```

Given that this repo doesn't have anything to do with the catalog repo, I'd choose a different name. Similar than the `nephio-example-packages` repo.
```suggestion
  name: oai-packages
```

---

yes, good point - unless we plan to move it to the catalog (which AIUI @arora-sagar does not want to do).